setup/scripts/python/ldqm_db/amcmanager.py

```diff
@@ -5,18 +5,13 @@
 
 class AMCmanager:
   def __init__(self):
+    self.connection_file = "file://${GEM_ADDRESS_TABLE_PATH}/connections_ch.xml"
```

Comment:
We'll need to be able to pass this in as well, so I'll have to finalize that part for the different teststands